src/OpalCompiler-Core/RBMethodNode.extension.st

```diff
@@ -73,6 +73,15 @@ RBMethodNode >> generateIR [
 	^ self ir: ir
 ]
 
+{ #category : #'*OpalCompiler-Core' }
+RBMethodNode >> generateMethod [
```

Comment:
I think this change goes also in a bad direction.
I know this is an extension method, but this should be in the compiler, and the AST should not know the `compilationContext`

Reply:
It's again not a new method. See #generate and #generateWithSource. I just renamed it to be more descriptive.
But generally I agree with you.

Reply:
It's again not a new method. See #generate and #generateWithSource. I just renamed it to be more descriptive.
But generally I agree with you.